pom.xml

```diff
@@ -110,19 +113,21 @@
     <log4j.test.version>2.17.2</log4j.test.version>
     <slf4j.version>1.7.30</slf4j.version>
     <joda.version>2.9.9</joda.version>
-    <hadoop.version>2.10.1</hadoop.version>
+    <hadoop.version>3.1.0</hadoop.version>
```

Comment:
I can't remember if its a conscious decision that we already discussed, but whats the reason for not upgrading hadoop to 3.2.1 instead ?

Reply:
During our community syncs we had agreed on targeting hadoop 3.1.0 as this is officially what Hive supports as seen from its pom https://github.com/apache/hive/blob/master/pom.xml#L133.

I can internally sync with you on the many conflicts between supporting the hadoop and hive versions but some conflicts around protobuf and guava can be found in these JIRA from hive community. https://issues.apache.org/jira/browse/HIVE-24484?jql=project%20%3D%20HIVE%20AND%20text%20~%20%22upgrade%20Hadoop%22